Library/Homebrew/formula.rb

```diff
@@ -2273,7 +2283,7 @@ def generate_completions_from_executable(*commands,
     base_name ||= name
 
     completion_script_path_map = {
-      bash: bash_completion/base_name,
+      bash: bash_completion(compat: shell_parameter_format != :click)/base_name,
```

Comment:
Pulling this into a variable with a comment would be nice; not clear why this is the case and a little hard to read the logic inline the method call